tests/openfl/communication/test_envoys_client.py

```diff
@@ -21,6 +26,7 @@ def director_client(director_pb2_grpc):
 
 
 def test_repotr_shard_info(director_client):
+    """Test repoer shard info RPC."""
```

Comment:
report? test_**repotr**_shard_info and """Test **repoer** shard info RPC."""

Reply:
fixed